ocproxy/api/api.go

```diff
@@ -622,6 +631,15 @@ func (p *proxy) onlyOfficeTrackInternal(w http.ResponseWriter, r *http.Request,
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+
+		if (req.Status == trackerStatusMustSave) { // + trackerStatusCorrupted + trackerStatusForceSavingError ???
```

Comment:
For what I see, `Corrupted` could fit in the same category as `MustSave`.  Can't really find anything for `ForceSavingError`.

Reply:
OK, in fact it could be enough to (re)interpret https://api.onlyoffice.com/editors/callback:
* `trackerStatusForceSavingError` = "7 error has occurred while force saving the document.". We should NOT unlock NOR save (at the moment we do save, it doesn't seem correct to me).
* `trackerStatusCorrupted` = "3 - document saving error has occurred". Same as above.

Will ask for confirmation from OnlyOffice, but from the above, we should save AND unlock with the `trackerStatusEditing` and `trackerStatusMustSave` only.

Reply:
Correction: it is OK to save in status 3 and 7 (from OO support) - and then unlock as well. Corruptions are actually handled here:
https://github.com/cernbox/onlyoffice-owncloud/blob/reva-oo-evaluation/controller/callbackcontroller.php#L334